zap_log_test.go

```diff
@@ -249,3 +250,15 @@ func (t *testZapLogSuite) TestRotateLog(c *C) {
 	c.Assert(len(files), Equals, 2)
 	os.RemoveAll(tempDir)
 }
+
+func (t *testZapLogSuite) TestErrorLog(c *C) {
+	conf := &Config{Level: "debug", File: FileLogConfig{}, DisableTimestamp: true}
+	lg := newZapTestLogger(conf, c)
+	lg.Error("", zap.NamedError("err", mockStackMethod()))
```

Comment:
why don't we use errors.New("log-stack-test") directly wince mockStackMethod is only used here?

Reply:
ok, I will remove that. I just want add a "error-stack-frame", emm we can remove that.